peering/forms.py

```diff
@@ -707,6 +707,11 @@ class RouterForm(BootstrapMixin, forms.ModelForm):
     )
     comments = CommentField()
     tags = TagField(required=False)
+    
+    napalm_username = forms.CharField(required=False, label="Username")
+    napalm_password = forms.CharField(required=False, label="Password")
+    napalm_timeout = forms.IntegerField(label="Timeout in seconds", initial=0)
+    napalm_args = forms.CharField(required=False, label="Optional Arguments (dictionary)")
```

Comment:
It could be useful to have a link to the NAPALM documentation for this field inside its helper text.

Reply:
See the help_texts dictionary, but i don't know how to "activate" them. If you tell me how to do that, I will do that.

Reply:
Could you gvie me a documentation link or a example in your code where you display the help text?

Reply:
Help texts for form fields can be defined in the `help_texts` dictionary of form `Meta` classes. They can also be defined right in the field attribute of a model by using the `help_text` named parameter of a field constructor.

Reply:
What I have tried is:
`napalm_timeout = models.PositiveIntegerField(blank=True, default=30, help_text="Test")`
in models.py, made a migration (it altered the field), but nothing changed.
And then I've tried adding it to the help_text dict in forms.py:```

    class Meta:
        model = Router

        fields = (
            "netbox_device_id",
            "use_netbox",
            "name",
            "hostname",
            "napalm_username",
            "napalm_password",
            "napalm_timeout",
            "napalm_args",
            "platform",
            "encrypt_passwords",
            "configuration_template",
            "comments",
            "tags",
        )
        labels = {"use_netbox": "Use NetBox", "configuration_template": "Configuration"}
        help_texts = {
            "hostname": "Router hostname (must be resolvable) or IP address",
            "configuration_template": "Template used to generate device configuration",
            "napalm_timeout": "Maximum time to wait for a connection (in seconds)",
            "napalm_args": "see .. for more",
```
Still no help_text.
Im kinda stuck now... 😅